backend/lambda_functions/interview_count_lambda.py

```diff
@@ -0,0 +1,80 @@
+import json
+import boto3
+import requests
+from datetime import datetime, timedelta
+
+cloudwatch = boto3.client('cloudwatch')
+
+def lambda_handler(event, context):
+    # we need to Define the real CloudWatch metric to retrieve
+    metric_name = 'InterviewCount'
+    namespace = 'Interviews'
+    company_name = event.get('company_name')
+    interview_date = event.get('interview_date')
+    
+    if not company_name or not interview_date:
```

Comment:
This is good, but we want to be able to pull all existing metrics without specifying the `date` and `company name`.
Please add the ability to pull all existing metrics.